yb-voyager/cmd/analyzeSchema.go

```diff
@@ -555,6 +556,7 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			}
 			for _, partitionColumn := range partitionColumnsList {
 				if !slices.Contains(primaryKeyColumnsList, partitionColumn) { //partition key not in PK
+					summaryMap["TABLE"].invalidCount[sqlInfo.objName] = true
```

Comment:
can you please add this to the above two `reportCase()` occurrences for `TABLE` in this else-if block.

Reply:
done, thanks